### ISSUE TITLE
(Seemingly) random crashes on MacOS

### ISSUE THREAD

**Issue by MatthewBennington** [runelite/runelite#13408]
**Describe the bug**
Runelite closes unexpectedly. I have not been able to identify a pattern.

**To Reproduce**
Can not reproduce manually yet.

**Additional context**
My log file is 1142 lines long, so I'm not including it in the body of this issue. You can find it as an attachment.
[jvm_crash_pid_4315.log](https://github.com/runelite/runelite/files/6223374/jvm_crash_pid_4315.log)

**Comment by MatthewBennington** [comment]
Another crash - 
[jvm_crash_pid_521.log](https://github.com/runelite/runelite/files/6251223/jvm_crash_pid_521.log)

**Comment by GeorgeKosrs** [comment]
I've been told its an AMD driver issue so I am glad its happening to other mac users who are not using AMD. It's interesting yours are like mine which are 20 hours+

Please keep reporting these crashes so we can get to the bottom of this.

Hooder has been helping me behind the scenes to try figure this out when he is not busy. He has let me know a solution is not probable but we can hope.

I know @IComplainInComments and @cgurps were coding a external plugin that uses the metal api. Maybe that is our only option if that ever happens.

**Comment by MatthewBennington** [comment]
[jvm_crash_pid_7904.log](https://github.com/runelite/runelite/files/6261204/jvm_crash_pid_7904.log)

[crash_details.txt](https://github.com/runelite/runelite/files/6261205/crash_details.txt)

**Comment by alexandrabush** [comment]
This has happened to me 3 times now. I can't see any pattern with it. The last time this happened I had just teleported to the farming guild and had started running to the shamans area. I've attached the 3 crash reports in a zip here.

Environment:
OS: macOS Mojave 10.14.6, 1.2 GHz Intel Core m3 
Runelite version: 1.7.15
Launcher version: 2.0.0

[Runelite Mac Crash Reports.zip](https://github.com/runelite/runelite/files/6796238/Runelite.Mac.Crash.Reports.zip)
